### PR TITLE
feat: add rule prefer-inline-action-props

### LINKED ISSUE #66: New rule: action prop name
Ref: https://github.com/ngrx/platform/issues/2892

### PR DESCRIPTION
**Pull request by rafaelss95**
Closes #66.

### CODE REVIEW

File: src/rules/action-prop-name.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+export const ruleName = 'action-prop-name'
+
+export const actionPropName = 'actionPropName'
+export const actionPropNameSuggest = 'actionPropNameSuggest'
+export type MessageIds = typeof actionPropName | typeof actionPropNameSuggest
+
+type Options = []
+
+export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
```
Comment (rafaelss95) [inline src/rules/action-prop-name.ts:14]: Let me know what do you think about the rule name and its description/failures.

Comment (timdeschryver) [reply src/rules/action-prop-name.ts:14]: What do you think about prefer-inline-action-props ?

**Comment by timdeschryver** [comment]
@rafaelss95 thanks for the contribution 🙂
I just wanted to check if you agree with the rename to `prefer-inline-action-props`. If so, do you have to make the changes? Otherwise, I could also do them.

**Comment by rafaelss95** [comment]
Hey @timdeschryver don't worry, I'll do it asap.

File: tests/rules/action-prop-name.test.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  ruleName,
+} from '../../src/rules/action-prop-name'
+import { ruleTester } from '../utils'
+
+ruleTester().run(ruleName, rule, {
+  valid: [
+    `const ok0 = createAction('ok0', props<{ id: number, name: string }>())`,
+    `const ok1 = createAction('ok1', props<Readonly<{ description: string }>>())`,
+    `const ok2 = createAction('ok2', props<Readonly<HttpErrorResponse & { description: string }>>())`,
+    `const saveUserSuccess = createAction('[API/User] Save user success', (user: User<number>) => ({
```
[the remark this answers is not in the document]
Comment (rafaelss95) [reply tests/rules/action-prop-name.test.ts:13]: Good catch, I'll add it soon.

**Comment by timdeschryver** [comment]
Take your time, there's no rush @rafaelss95 🙂
I didn't mean to rush you.

**Comment by rafaelss95** [comment]
> Take your time, there's no rush @rafaelss95 🙂
> I didn't mean to rush you.

Ah, nvm, that's fine 😄 
I didn't do it yesterday because I was with a lot of work.

I renamed it to `prefer-inline-action-props` and also added the missing test case you mentioned. Please take a look and let me know if it's everything fine now.

File: src/rules/action-prop-name.ts (hunk 1 above)
Comment (rafaelss95) [reply src/rules/action-prop-name.ts]: Done. What about the description and failure messages?

**Review by timdeschryver — APPROVED** [review]

File: src/rules/action-prop-name.ts (hunk 1 above)
Comment (timdeschryver) [reply src/rules/action-prop-name.ts]: I think it's good.  👍
We'll provide more info in the docs folder soonish

**Pull request merged**